openstack/input.yaml

```diff
@@ -28,4 +27,19 @@ modak_api_env:
   MODAK_DATABASE_PORT: "3306"
 modak_db_env:
   MYSQL_ROOT_HOST: "%"
-  MYSQL_ROOT_PASSWORD: password
+vault_env:  
+  VAULT_LOCAL_CONFIG: '{"backend": {"file": {"path": "/vault/file"}}, "default_lease_ttl": "16h", "max_lease_ttl": "72h"}'
+  VAULT_DEV_ROOT_TOKEN_ID: test_token
+  VAULT_TOKEN: test_token
+keycloak_env:
+  KEYCLOAK_USER: admin
+  KEYCLOAK_PASSWORD: admin
+  KEYCLOAK_IMPORT: /tmp/sodalite-realm.json
+keycloak_client_secret: ca4d390e-c1ac-4a1d-afd7-51c1e9e16c6f
+vault_token: test_token
+project_domains: [ snow, clinical, vehicle, pds ]
+pds_env:
+  SECRET_VAULT_LOGIN_URI: http://vault:8200/v1/auth/jwt/login 
+  SECRET_VAULT_URI: http://vault:8200/v1/ 
+  OIDC_INTROSPECTION_ENDPOINT: http://keycloak:8080/auth/realms/SODALITE/protocol/openid-connect/token/introspect 
+  OIDC_CLIENT_SECRET: ca4d390e-c1ac-4a1d-afd7-51c1e9e16c6f
```

Comment:
I hope this secret is not valid

Reply:
Well it can be valid as any other GUID. Just an example how secret should look like.

Reply:
You are right :)